ovos_dinkum_listener/voice_loop/voice_loop.py

```diff
@@ -286,9 +287,9 @@ def stop_recording(self):
 
     def _in_recording(self, chunk: bytes):
         """
-        Handle a chunk of audio as user input
-        (after wake word detection or in continuous listening mode). Check for
-        hotwords in all cases and pass audio to any loaded audio transformers.
+        Handle a chunk of audio as user input while in the `RECORDING` state
+        (i.e. after wake word detection). Check for hotwords in all cases and
```

Comment:
clarify it checks for stop words only

unlike sleep mode this does not require a previous wake word detection. It is meant as a failsafe to ensure you can always exit recording mode for devices near you